src/Backend/Modules/Analytics/Actions/AddLandingPage.php

```diff
@@ -111,7 +112,10 @@ private function validateForm()
             if (!isset($page)) {
                 $this->frm->getField('page_path')->addError(BL::err('FieldIsRequired'));
             }
-            if (!$this->frm->getField('page_path')->isFilled() && !$this->frm->getfield('page_list')->isFilled()) {
+            if (!$this->frm->getField('page_path')->isFilled()
+                && !empty($this->linkList)
```

Comment:
You can als ouse `$this->frm->hasField('page_path')`. This seems more logical to me (since this doesn't cause duplicate code in the form of the check on the $this->linkList variable.